lib/usage.js

```diff
@@ -28,17 +28,22 @@ module.exports = function (yargs, y18n) {
     return self
   }
 
+  var outputFailure = true
```

Comment:
I thought of that, but then I noticed `showHelpOnFail` above defaults to `true`, so I figured it depended on the use-case. I also thought `if (outputFailure)` was easier to read/follow than `if (!something) `. But I'm happy to change it if you'd like.